clients/src/main/java/org/apache/kafka/common/requests/AbstractRequest.java

```diff
@@ -303,6 +303,10 @@ private static AbstractRequest doParseRequest(ApiKeys apiKey, short apiVersion,
                 return ListTransactionsRequest.parse(buffer, apiVersion);
             case ALLOCATE_PRODUCER_IDS:
                 return AllocateProducerIdsRequest.parse(buffer, apiVersion);
+            case GET_TELEMETRY_SUBSCRIPTION:
```

Comment:
Same here as elsewhere: should subscription be singular or plural. I don't care, but let's check with @edenhill.